### PR TITLE
Refactor/change position of tests

### PR DESCRIPTION
**Pull request by patrick-tolosa**
Notes for reviewer:

This PR is chained off #76 
This is pending approval of #76 .

This PR refactors some of our tests, with the idea of putting tests into the right files (instead of one file per test)

### CODE REVIEW

**Comment by asisayag2** [comment]
@patrick-tolosa , didn't get the change that was done here.

**Comment by patrick-tolosa** [comment]
@asisayag2  - 

All our tests were previous written in a flat list inide a single folder.

All tests were broken down to seprate files even when touching the same area, for example we had these files as tests with a single describe/it in them:

video+transformation
video+poster
video+poster+transformation

This PR deals with two things:

1. Group similar tests to a single file (for example video.spec instead of multiple video files)
2. Group tests into folders instead of having a flat list of siblings.

**Comment by asisayag2** [comment]
@patrick-tolosa , but I see more changes in there... added lazy loading + some functions that were modified, not just re-located

File: bundlewatch.config.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ const bundlewatchConfig = {
   files: [
     {
       path: './dist/Cloudinary.umd.js',
-      maxSize: '12kb'
+      maxSize: '14kb'
```
[the remark this answers is not in the document]
Comment (patrick-tolosa) [reply bundlewatch.config.js:5]: Because this PR is chained off #76 (Placeholder)

**Pull request merged**